setup.py

```diff
@@ -21,10 +21,14 @@
     name='nb2kg',
     author='Jupyter Development Team',
     author_email='jupyter@googlegroups.com',
-    description='Extension for Jupyter Notebook 4.2.x to enable remote kernels hosted by Kernel Gateway',
-    long_description = '''See `the README <https://github.com/jupyter/kernel_gateway_demos/nb2kg>`_ for more information.
+    description='Extension for Jupyter Notebook 4.2.x to enable remote kernels hosted by Kernel Gateway or Enterprise Gateway',
```

Comment:
Should we say 4.2.x and 5.x ?

Reply:
@lresende, I guess I've always assumed that `4.2.x` implies `>= 4.2` - which includes `5.x`.  The only real requirement is that it be used with a version of Notebook that supports extensions - thus >= 4.2.